admin/deploy/templates/admin.deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aro-hcp-admin
+  namespace: '{{ .Release.namespace  }}'
+  labels:
+    app: aro-hcp-admin
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: aro-hcp-admin
+  template:
+    metadata:
+      labels:
+        app: aro-hcp-admin
+    spec:
+      containers:
+        - name: aro-hcp-admin
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
please use digest based image references instead of tag based ones.
frontend/backend decided on using a `deployment.imageName` helm variable used in the deployment

```
image: '{{ .Values.deployment.imageName }}'
```

.. which is constructured in the Makefile as a digest based image reference